src/LiveStorage.test.ts

```diff
@@ -0,0 +1,8 @@
+import { LiveStorage } from './LiveStorage'
```

Comment:
May I suggest prefixing this test module with `.node.test.ts`? 

There will also be browser tests, and it seems there's a convention we may adopt from MSW to use `.test.ts` for browser tests, and `.node.test` for those that run in NodeJS.

Reply:
updated